### ISSUE TITLE
Messagebox placement error

### ISSUE THREAD

**Issue by boleang** [Akascape/CTkMessagebox#63]
Hi! I've been using your messagebox for my project. At the beginning, everything worked just fine. But recently, the messagebox gotten into sort of error, like this:

```
Exception` in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python3.9/tkinter/__init__.py", line 1892, in __call__
    return self.func(*args)
  File "/home/pi/.local/lib/python3.9/site-packages/CTkMessagebox/ctkmessagebox.py", line 428, in move_window
    self.y = event.y_root - self.oldy
AttributeError: 'CTkMessagebox' object has no attribute 'oldy'

```
I'm using your messagebox to show error message if my realtime data fetcher function catches empty data. Again, the error just recently showing, no error from before. That error resulting on the messagebox pop-up that usully shown in the middle of my desktop and stacked only in that spot (in my case, i could have more than 1 messagebox to be shown at one time). But due the error, those messagebox pop-ups shown at random spots in my desktop, and sometimes the button (both "X" and my custom buttons) cant be clicked, restraining to continue the next action (such closing the messagebox)

**Comment by Akascape** [comment]
fixed in v2.6, update

**Comment by boleang** [comment]
> fixed in v2.6, update

Thank you so much!
I actually found the "solver" a day after i sent this issues. Cant explain it in detail, but i use a function to write a message (as string) and passing the message as argument (as this function is the one that the fetcher would call). Then another function to pass that argument using after method : self.after(0, self.show_message, message) instead directly passing the arguments from the fetcher. And last the function show_message(self, message to finally call your custom message box by using the message arguments as the text.

But now i'll try the update, your message box is a lifesaver for my college last project and i love it too!
